src/Omega_h_mesh.hpp

```diff
@@ -127,92 +127,113 @@ class Mesh {
   Graph ask_star(Int dim);
   Graph ask_dual();
 
-/** ask_revClass (Int edim, LOs class_ids): takes input of entity dimension
- * 'edim', and an 1d array of model entity IDs to return
- * a CSR structure (Adj) containing IDs of mesh entities classified on the 
- * requested input model entities. Note here that 'edim' is equal to the
- * model entity dimension as well as the dimension of returned mesh entities
- * NOTE: if the model entity is a region, the memory usage is high
- */
-  Adj ask_revClass (Int edim, LOs class_ids);
-
-/** ask_revClass (Int edim): see ask_revClass (Int edim, LOs class_ids) above.
- * Here, the output is for all model entities of dimension 'edim' instead
- * of a input list
- */
-  Adj ask_revClass (Int edim);
-
-/** ask_revClass_downAdj (Int from, Int to): takes input of a higher
- * dimension 'from' and a lower dimension 'to'. The value of 'from' is equal
- * to the mesh and model entity dimensions used to get reverse class.
- * (similar to 'edim' for ask_revClass functions above.) This function can be
- * understood as a two step process. Firstly, for all the model entities
- * of dimension 'from', we get the reverse classified mesh entities.
- * Then combine the reverse classification, and downward adjacency information.
- * The final output is a CSR containing downward adjacent
- * mesh entities of dimension 'to' which bound the reverse classified mesh
- * entities of dimension 'from'.
- */
-  Adj ask_revClass_downAdj (Int from, Int to);
-
-/** has_revClass (Int edim): Input is a entity dimension 'edim'. This function
- * checks if the reverse classification for that dimension is present in
- * memory or not.
- */
-  bool has_revClass (Int edim) const;
-
-/** Takes input of model entity IDs, entity dimension, name of field and number
- * of components, to create a the rcField. This function
- * is used when fields are to be stored with mesh entities returned by
- * ask_revClass (Int edim, LOs class_ids)
- */
+  /** ask_revClass (Int edim, LOs class_ids): takes input of entity dimension
```

Comment:
is this fixing formatting?

Reply:
I'm guessing my editor inserted this based on the .clang-format but it definitely looks weird.

I can go through and remove all this added white space if you think I should.